docs/misc/changelog.rst

```diff
@@ -18,6 +18,7 @@ New Features:
 ^^^^^^^^^^^^^
 - Add `n_cpu_tf_sess` to model constructor to choose the number of threads used by Tensorflow
 - `VecNormalize` now supports being pickled and unpickled.
+- Add parameter `exploration_initial_eps` to DQN to mirror `exploration_final_eps`, allowing user to start training with random action probability which is not 1 (default remains 1)
```

Comment:
You can add your nick here in the end, and also remember to update contributors list at the end of the file.